CMSgov/easi-app
# [EASI-3839] ip info

# EASI-3839


## Changes and Description

- Returns `ip` information in `SystemMaintainerInformation`
- Adds mock data for both `SystemMaintainerInformation` and `BusinessOwnerInformation`



## How to test this change

- pull branch, open postman, and import the EASI collection
- the new request is in `CEDAR/core/System Details/Get System Details`
- test run the request (make sure CEDAR is set up to mock data locally)
- confirm data comes back for System Maintainer and Business Owner



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
